bitnami/metallb/templates/controller/deployment.yaml

```diff
@@ -110,8 +116,15 @@ spec:
           ports:
             - name: metrics
               containerPort: {{ .Values.controller.containerPorts.metrics }}
+            - name: webhook-server
+              containerPort: 9443
+              protocol: TCP
+          volumeMounts:
+            - name: cert
+              mountPath: /tmp/k8s-webhook-server/serving-certs
+              readOnly: true
           {{- if .Values.controller.extraVolumeMounts }}
-          volumeMounts: {{- include "common.tplvalues.render" (dict "value" .Values.controller.extraVolumeMounts "context" $) | nindent 12 }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.controller.extraVolumeMounts "context" $) | nindent 12 }}
           {{- end }}
```

Comment:
It would be nice if you could indent this to be aligned with the other elements in the array.